packages/dx-react-scheduler/src/plugins/month-view.jsx

```diff
@@ -72,9 +80,20 @@ export class MonthView extends React.PureComponent {
     this.viewCellsData = getters => computed(
       getters, viewName, this.viewCellsDataComputed, getters.viewCellsData,
     );
+
+    this.timeTableElement = getters => computed(
+      getters, viewName, this.timeTableElementComputed, getters.timeTableElement,
+    );
+    this.layoutElement = getters => computed(
+      getters, viewName, this.layoutElementComputed, getters.layoutElement,
+    );
+    this.layoutHeaderElement = getters => computed(
+      getters, viewName, this.layoutHeaderElementComputed, getters.layoutHeaderElement,
+    );
   }
 
   timeTableRef(timeTableRef) {
```

Comment:
This method is quite strange. According to its name it is used as a value for a *ref* property. It then can be replaced with a `React.createRef()` instance.

```js
// in constructor
this.timeTableRef = React.creareRef();
this.timeTable = { ref: this.timeTableRef };

// in a place where `this.state.timeTableRef` is used
this.timeTableRef.current...
```

Reply:
We want to update the component after change `this.state.timeTableRef`